### ISSUE TITLE
JaxpServiceResponseUnmarshaller throws an error when test returns something besides assertions

### ISSUE THREAD

**Issue by rjrudin** [marklogic-community/marklogic-unit-test#128]
To reproduce, try a multi-statement XQuery test where the first statement returns a string, and the second statement returns assertions that pass. JaxpServiceResponseUnmarshaller will blow up on the response due to line 71:

    Element resultNode = (Element) resultNodes.item(j);

Since the text node is not an Element, an exception is thrown.

I think the code should just ignore any node that is not an Element.

**Comment by venuiyengar** [comment]
Do you have an example that I can use to test the failure? Is the assert that is failing?

**Comment by rjrudin** [comment]
Just make a test that looks like this:

```
... some xquery code ...
return "Hi";

... some test code ...
return test:assert-equal(...)
```

**Comment by venuiyengar** [comment]
I have a fix for this should I push the changes as part of the same pull request?

**Comment by rjrudin** [comment]
Always do a separate PR per issue. Keeps the history clean.

**Comment by rjrudin** [comment]
Addressed via #132
